docs/how-to/detect-ipfs-on-web.md

```diff
@@ -0,0 +1,127 @@
+# Detect IPFS on the Web
```

Comment:
Title should probably be more general, because is not just about detection. eg "Handling IPFS Content in Web User Agents" or something in that direction.

Reply:
We already have "Addressing IPFS on the Web" so I went with similar title, but not feeling strong either way.